Mathlib/MeasureTheory/Integral/Periodic.lean

```diff
@@ -221,14 +226,61 @@ protected theorem intervalIntegral_preimage (t : ℝ) (f : UnitAddCircle → E)
 
 end UnitAddCircle
 
-variable {E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
+
+/-!
+## Interval integrability of periodic functions
+-/
 
 namespace Function
 
 namespace Periodic
 
+variable {E : Type*} [NormedAddCommGroup E]
+
 variable {f : ℝ → E} {T : ℝ}
 
+/-- A periodic function is interval integrable over every interval if it is interval integrable
+over one period. -/
+theorem intervalIntegrable {t : ℝ} (h₁f : Function.Periodic f T) (hT : 0 < T)
```

Comment:
Do you actually need `T` to be positive? (How do other theorems deal with this?)

Reply:
I guess periodicity can be used formulated and used for negative `T`. However, it can not see a single theorem that does that. In this file alone, the assumption `(hT : 0 < T)` seems to appear 11 times.

My instinct is to leave the assumption as it is. If you disagree, I will weaken the hypothesis, but then I feel this should also be done for the other theorems -- which amounts to a massive refactor. Please advise.

Best -- Stefan.

Reply:
I guess periodicity can be formulated and used for negative `T`. However, I could not find a single theorem that does that -- the assumption `0 < T` appears 11 times in this file alone. I wonder if it makes sense to have a generalized assumption here when it does not appear in any of the other theorems. Please advise.

Reply:
Good point, thanks for checking. In this case, I'd go with what the other theorems do. (If another reviewer has strong opinions, we can always reconsider. I don't mind keeping it as-is.)